spec/cypress/cypress_helper.rb

```diff
@@ -0,0 +1,33 @@
+# This is loaded once before the first command is executed
+
+begin
+ require 'database_cleaner'
+rescue LoadError => e
+ puts e.message
+end
+
+begin
+  require 'factory_bot_rails'
+rescue LoadError => e
+  puts e.message
+  begin
+    require 'factory_girl_rails'
+  rescue LoadError => e
+    puts e.message
+  end
+end
+
+require 'cypress_on_rails/smart_factory_wrapper'
+
+factory = CypressOnRails::SimpleRailsFactory
+factory = FactoryBot if defined?(FactoryBot)
+factory = FactoryGirl if defined?(FactoryGirl)
```

Comment:
Again, let's just use the one we want.

Reply:
;) Aaa thoughtworks FactoryGirl - a great example of removing sexism from the code - quite an interesting case study